contrib/epee/include/serialization/keyvalue_serialization.h

```diff
@@ -108,6 +108,9 @@ public: \
 #define KV_SERIALIZE_CONTAINER_POD_AS_BLOB(varialble)     KV_SERIALIZE_CONTAINER_POD_AS_BLOB_N(varialble, #varialble)
 #define KV_SERIALIZE_OPT(variable,default_value)          KV_SERIALIZE_OPT_N(variable, #variable, default_value)
 
+#define KV_SERIALIZE_ENTRY_FIELD_IF_REQUESTED(var) \
```

Comment:
I would define this macro next to the usage code since I don't think this is going to be used outside of this feature in particular and helps readability.

Reply:
updated